services/web-app/pages/assets/[host]/[org]/[repo]/[library]/[ref]/index.module.scss

```diff
@@ -35,3 +38,18 @@
     width: 50%;
   }
 }
+
+.resources-section {
+  h5 {
+    @include type.type-style('body-compact-01');
+
+    color: theme.$text-secondary;
+  }
+
+  h4 {
+    @include type.type-style('body-compact-02');
+    @include mixins.line-clamp(2);
+
+    color: theme.$text-primary;
+  }
+}
```

Comment:
This seems like it might be something we should add to the component. I'm going to check with design. https://ibm-studios.slack.com/archives/C022V4871K2/p1655499056464809?thread_ts=1655312718.200569&cid=C022V4871K2